src/Shop.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import { Card, Row, Jumbotron, Container } from 'react-bootstrap'
+import MySpinner from './MySpinner'
+
+export default function Shop() {
+    const FortniteAPI = require("fortnite-api-io");
+    const fortniteAPI = new FortniteAPI("ef976a9b-a4686e4c-53d900e4-77767b89")
+    
+    const [items, setItems] = useState([])
+    const [loading, setLoading] = useState(true)
+
+    const fetchItems = async () => {
+        const upcomingItems = await fortniteAPI.listUpcomingItems();
+        setItems(upcomingItems.items)
+        setLoading(false)
+        console.log(upcomingItems.items)
+    }
+
+    useEffect(() => {
+        fetchItems();
+    }, [])
+
+
+    const itemHtml = items.map((item) => {
+        let classes = [];
+        if (item.rarity === 'rare') classes.push("rare");
```

Comment:
ah, so you're copying and pasting code here. this is usually a good sign that you pull this code out into a separate component, or at the least, you can probably pull this out into a simple utility method.